INSTALL.md

```diff
@@ -4,6 +4,7 @@
 * [Get the source code](#get-the-source-code)
 * [Dependencies](#dependencies)
   * [Linux dependencies](#linux-dependencies)
+* [Building Skia dependency](#building-skia-dependency-os-xlinuxbsd)
```

Comment:
>  is linking to the installation instructions on the Skia website OK?

Yes, that's is what I think we should do. Linking to the official instructions. (In that way we don't need to keep them in sync)